airbyte-webapp/src/locales/en.json

```diff
@@ -183,6 +185,7 @@
 
   "connection.warningUpdateSchema": "WARNING! Updating the schema will delete all the data for this connection in your destination and start syncing from scratch.",
   "connection.fromTo": "{source} → {destination}",
+  "connection.itemHeadTitle": "Connection | {source} → {destination}",
```

Comment:
I think all logic for titles ( where to put `|` and where to put `→` ) should be part of UI instead of locales.

Could you please create a separate component for it? This will also make it less error-prone during adding new locales, as we won't need to duplicate Page names in steps (  `"admin.sourcesHeadTitle": "Admin | Sources"` and ` "admin.configurationHeadTitle": "Admin | Configuration"` )

Reply:
Taking into consideration @avaidyanatha 's comment, I am now even more confident, that we should commonize logic in 1 place instead of providing separator explicitly in `en.json`

Reply:
Good point. I refactored `HeadTitle` to support title chaining.